.typedoc/__tests__/__snapshots__/file-structure.test.ts.snap

```diff
@@ -67,7 +67,9 @@ exports[`Typedoc output > should have a deliberate file structure 1`] = `
   "types/user-organization-invitation-resource.mdx",
   "types/user-resource.mdx",
   "types/without.mdx",
+  "shared/api-url-from-publishable-key.mdx",
```

Comment:
Whenever we add JSDoc comments to previously undocumented functions we'll need to update this snapshot as Typedoc will only generate MDX files for documented, exported functions.

You can update the snapshot by running `pnpm test:typedoc -u`